osu.Game/Rulesets/UI/FrameStabilityContainer.cs

```diff
@@ -63,20 +67,17 @@ public FrameStabilityContainer(double gameplayStartTime = double.MinValue)
         /// </remarks>
         private int direction = 1;
 
-        [BackgroundDependencyLoader(true)]
-        private void load(IGameplayClock clock)
+        [BackgroundDependencyLoader]
+        private void load(IGameplayClock? gameplayClock)
```

Comment:
Are we still changing `[BackgroundDependencyLoader(true)]` to NRT-annotated params after [the latest mono-related fiasco](https://discord.com/channels/188630481301012481/188630652340404224/1006802093232881715)? Not that I imagine this particular one will cause issues, but it's a bit touch-and-go and I'm not sure where we're at regarding this.

Reply:
I did consider that for this specific usage, but I do believe it's only a problem in tests. Can revert if preferred.

Reply:
Should be okay. Honestly would like to see NET6 prioritised still...